docs/tctl/schedule/create.md

```diff
@@ -26,4 +26,19 @@ It starts a Workflow `YourWorkflowType` on Task Queue `your-task-queue`, giving
 Workflows do not be run in parallel.
 If they would other overlap, they are buffered to run sequentially.
 
+You can use traditiona cron strings also, including all of the features that are supposed by `CronSchedule` today,
+such as `@weekly` and other shorthands, `@every`, and `CRON_TZ`.
+
+```shell
+$ tctl schedule create \
+    --sid 'your-schedule-id' \
+    --cron '3 11 * * Fri' \
+    --wid 'your-workflow-id' \
+    --tq 'your-task-queue' \
+    --type 'YourWorkflowType'
+```
+
+Any combination of `--cal`, `--interval`, and `--cron` is supported and Actions will happen at any of the specified times.
+If you use both `--time-zone` and also `CRON_TZ`, they must agree.
```

Comment:
```suggestion
Any combination of `--cal`, `--interval`, and `--cron` is supported, and Actions can happen at any of the specified times.
If you use both `--time-zone` and `CRON_TZ`, they must agree.
```

Reply:
It's not that useful of a feature if we only promise that actions "can" happen... :)

Reply:
Entirely valid. I retract my edit.